portfolio/src/main/java/com/google/sps/servlets/DeletionServlet.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.commons.io.IOUtils;
+
+/** Servlet that returns a programmable number of comments */
+@WebServlet("/delete-comment")
+public class DeletionServlet extends HttpServlet {
+
+  /**
+    * POST handler for the /delete-comment route
+    */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
+    /* 
+     * Get the body of the request.
+     * bodyString must be a json encoded DeletionRequestBody, with shape similar to below
+     * {
+     *   "id": COMMENT_ID
+     * }
+     */
+    String deletionRequestJSON = IOUtils.toString(request.getReader());
+
+    // parse the json request body and create a DeletionRequestBody from its contents
```

Comment:
Try to make sure comments start with capitals and have proper punctuation.

Reply:
Fixed 👍 I'll make a quick pass at comments in other files to ensure consistency, in a 
 future pr https://github.com/thomasogara/step-repository/pull/34/commits/739a269d78c27bb8317e5f392354a3e840fd5737